esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -114,3 +147,86 @@ var RefreshButton = React.createClass({
     </button>;
   },
 });
+
+
+// Modified from react-json-table example code.
+var SelectTable = React.createClass({
+  
+  propTypes: {
+    rows: React.PropTypes.array.isRequired,
+    saveState: React.PropTypes.shape({
+      greyed: React.PropTypes.object.isRequired,
+      sort: React.PropTypes.any.isRequired,
+      reverse: React.PropTypes.bool.isRequired
+    }).isRequired,
+    header: React.PropTypes.bool.isRequired,
+    columns: React.PropTypes.array,
+    updateTableState: React.PropTypes.func.isRequired
+  },
+    
+  getInitialState: function(){
+    // We will store the sorted column and whether each row is greyed out
+    return {sort: this.props.saveState.sort, greyed: this.props.saveState.greyed, reverse: this.props.saveState.reverse};
+  },  
+  render: function(){
+    this.props.updateTableState(this.state);
+    var me = this;
+    // clone the rows
+    items = this.props.rows.slice();
+    
+    items = _.sortBy(items, function( item ){
```

Comment:
I believe you can actually do `_.sortBy(items, me.state.sort)` and it will automatically do the lookup.